CONTRIBUTING.md

```diff
@@ -15,3 +15,11 @@
 * No linting errors
 * Spaces (2), not tabs
 * Flow Type Annotations are included
+
+### Migrations
+* [Sequelize-Cli List of Commands](https://www.npmjs.com/package/sequelize-cli)
```

Comment:
I think this needs more detail - I like the list of resources, but as a new developer on this project, I don't know what I need to do in order to create a migration... when is one necessary?  

As well, let's add a few bullets to the section above:
```
* When schema updated are required, sequelize schema migrations are provided with instructions to run in the PR message
* When data migrations are required, sequelize data migrations are provided with instructions to run in the PR message
```